lattice/core/processors.py

```
@@ -139,56 +145,64 @@ def _attempt_process_row(self, row: pd.Series, fields_dict: Dict[str, Dict[str,
+                    result_dict[field] = field_values[field]
                 else:
                     logger.warning(f"Missing field '{field}' in chain response")
-                    result[field] = None
-                    
-            return result
-            
+                    result_dict[field] = None
+
+            return result_dict
+
         except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)